2021/athanasiadis2022weaklysupervised/openreview/packages.tex

```diff
@@ -0,0 +1,39 @@
+% Import packages here
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage{hyperref}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{graphicx}       % Allows including images
+\usepackage{amsmath}
+\usepackage{bm}             % bold and italics maths at the same time
+\usepackage{amssymb}        % the definition equal triangle symbol
+\usepackage{xcolor}         % colors
+\usepackage{hyperref}       % links
+\usepackage{babel} % test gia figure horizontally
+\usepackage{float} % fix position
+\usepackage{times}
+\usepackage{epsfig}
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{amsthm}
+\usepackage{subfig}
+\usepackage{subfigure}
+\usepackage[labelformat=simple]{subcaption}
```

Comment:
If you are importing `subfigure`, then do not import `subcaption` - these two doesn't play well with each other.